features/mbedtls/platform/inc/platform_mbed.h

```diff
@@ -20,3 +20,7 @@
 #if defined(DEVICE_TRNG)
 #define MBEDTLS_ENTROPY_HARDWARE_ALT
 #endif
+
+#if defined(DEVICE_CRYPTO)
```

Comment:
In the future we may create an acceleration HAL, but in the meantime we have dedicated mbed TLS acceleration. I think the symbol to enable mbed TLS *specific* acceleration should suggest it's mbed TLS specific in it's name - so I would suggest a preprocessor symbol of `DEVICE_MBEDTLS_HW_SUPPORT` or something similar with `_MBEDTLS_` in the name.

Open to other ideas.